templates/zerver/help/zulip-on-windows.md

```diff
@@ -0,0 +1,29 @@
+# Using Zulip on Windows
+
+Zulip has a Windows client, compatible with Windows versions 7, 8 and 10; it is identical in
+functionality to the Zulip web application. The client
+can be download through the [Zulip website](https://zulip.org/).
+
+The client is regularly updated and is a wrapper of the web application, but with enhanced
+notifications and can run in the background. The current version does not contain
+any significant features additional to those in the web application.
+
+!!! warn ""
+    **Note:** Since Zulip is no longer running on a browser, many external keyboard shortcuts
+    endemic to a browser will no longer work, for example, ALT-LEFT to go back to the previous page.
+
+## Initial setup
+### Connecting to a server
```

Comment:
I just noticed this was updated, sorry for the delay! I think you can replace `## Initial setup` with `## Connecting to a server` since there's no more sections in `## Initial setup`. Regardless, thanks for your hard work so far!